### PR TITLE
Tests with aspect and prettier config

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: commands/init/assembly/__tests__/as-pect-example.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Vec3 {
```
Comment (peterjah) [inline commands/init/assembly/__tests__/as-pect-example.spec.ts:1]: do we really need this ? maybe the url of the as-pect documentation is enough

Comment (Thykof) [reply commands/init/assembly/__tests__/as-pect-example.spec.ts:1]: this true our boilerplate are not really good as for now

https://github.com/massalabs/massa-sc-toolkit/issues/70

File: commands/init/assembly/__tests__/massa-example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import {
+  Address,
+  Context,
+  fromBytes,
+  Storage,
+  toBytes,
+  Args,
+} from '@massalabs/massa-as-sdk';
+
+describe('An other group of test', () => {
```
Comment (SebastienValla) [inline commands/init/assembly/__tests__/massa-example.spec.ts:16]:
```suggestion
describe('Testing the Storage and CallStack', () => {
```

File: commands/init/assembly/__tests__/massa-example.spec.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  });
+
+  test('Testing the Storage set', () => {
+    setStorage([]);
+    expect(fromBytes(Storage.get(toBytes('test')))).toBe('value');
+  });
+
+  test('Testing the Storage set', () => {
+    setStorage([]);
+    expect(Storage.has(toBytes('test'))).toBe(true);
```
Comment (SebastienValla) [inline commands/init/assembly/__tests__/massa-example.spec.ts:56]:
```suggestion
    expect(Storage.has(toBytes('test'))).toBeTruthy()
```

File: commands/init/assembly/__tests__/massa-example.spec.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  });
+
+  test('Testing the Storage set', () => {
+    setStorage([]);
+    expect(
+      Storage.hasOf(
+        new Address('A12E6N5BFAdC2wyiBV6VJjqkWhpz1kLVp2XpbRdSnL1mKjCWT6oR'),
+        toBytes('test'),
+      ),
+    ).toBe(true);
```
Comment (SebastienValla) [inline commands/init/assembly/__tests__/massa-example.spec.ts:66]:
```suggestion
    ).toBeTruthy()
```

**Review by SebastienValla — CHANGES_REQUESTED** [review]

**Review by SebastienValla — APPROVED** [review]

File: commands/init/assembly/__tests__/as-pect-example.spec.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  constructor(public x: f64 = 0, public y: f64 = 0, public z: f64 = 0) {}
+}
+describe('example', () => {
+  it('should be 42', () => {
+    expect<i32>(19 + 23).toBe(42, '19 + 23 is 42');
```
Comment (gregLibert) [inline commands/init/assembly/__tests__/as-pect-example.spec.ts:6]:
```suggestion
    expect(19 + 23).toBe(42, '19 + 23 is 42');
```

File: commands/init/assembly/__tests__/as-pect-example.spec.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  });
+
+  it('should be the same reference', () => {
+    const ref = new Vec3();
+    expect<Vec3>(ref).toBe(ref, 'Reference Equality');
```
Comment (gregLibert) [inline commands/init/assembly/__tests__/as-pect-example.spec.ts:11]: I'm not sure we need the <Vec3> part

File: commands/init/assembly/__tests__/as-pect-example.spec.ts [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+  it('should compare values', () => {
+    expect<i32>(10).toBeLessThan(200);
+    expect<i32>(1000).toBeGreaterThan(200);
+    expect<i32>(1000).toBeGreaterThanOrEqual(1000);
+    expect<i32>(1000).toBeLessThanOrEqual(1000);
+  });
+
+  it('can log some values to the console', () => {
+    log<string>('Hello world!'); // strings!
```
Comment (gregLibert) [inline commands/init/assembly/__tests__/as-pect-example.spec.ts:36]: Do we need the <string> part ?

File: commands/init/assembly/__tests__/massa-example.spec.ts [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+import {
+  Address,
+  Context,
+  fromBytes,
+  Storage,
+  toBytes,
+  Args,
+} from '@massalabs/massa-as-sdk';
+
+describe('Testing the Storage and CallStack', () => {
```
Comment (gregLibert) [inline commands/init/assembly/__tests__/massa-example.spec.ts:16]: I think that using checksForEachLineThatThe would make all those tests shorter

File: commands/init/assembly/contracts/hello-world.ts [hunk 9]
```diff
@@ -1,6 +1,6 @@
-import { generateEvent } from "@massalabs/massa-as-sdk";
+import { generateEvent } from '@massalabs/massa-as-sdk';
 
 export function main(_: StaticArray<u8>): StaticArray<u8> {
-    generateEvent("hello world");
-    return [];
+  generateEvent('hello world');
```
Comment (gregLibert) [inline commands/init/assembly/contracts/hello-world.ts:4]:
```suggestion
  generateEvent('Hello, World!');
```

File: commands/init/assembly/contracts/main.ts [hunk 10]
```diff
@@ -1,17 +1,38 @@
...
 
-const keyTest = new Args().add("test").serialize();
-const valueTest = new Args().add("value").serialize();
+const keyTestArgs = new Args().add('test');
+const valueTestArgs = new Args().add('value');
+
+const keyTest = toBytes('test');
+const valueTest = toBytes('value');
+
+export function setStorageOf(_args: StaticArray<u8>): StaticArray<u8> {
```
Comment (gregLibert) [inline commands/init/assembly/contracts/main.ts:20]:
```suggestion
export function setStorageOf(_: StaticArray<u8>): StaticArray<u8> {
```

File: commands/init/assembly/contracts/main.ts [hunk 11]
```diff
@@ -1,17 +1,38 @@
...
+  Storage.setOf(testAddress, keyTest, valueTest);
+  return valueTest;
+}
+
+export function setStorageOfWithArgs(_args: StaticArray<u8>): StaticArray<u8> {
```
Comment (gregLibert) [inline commands/init/assembly/contracts/main.ts:25]:
```suggestion
export function setStorageOfWithArgs(_: StaticArray<u8>): StaticArray<u8> {
```

File: commands/init/assembly/contracts/main.ts [hunk 12]
```diff
@@ -1,17 +1,38 @@
...
+  Storage.setOf(testAddress, keyTestArgs, valueTestArgs);
+  return valueTestArgs.serialize();
+}
 
 export function setStorage(_args: StaticArray<u8>): StaticArray<u8> {
```
Comment (gregLibert) [inline commands/init/assembly/contracts/main.ts:30]:
```suggestion
export function setStorage(_: StaticArray<u8>): StaticArray<u8> {
```

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Comment by Thykof** [comment]
@gregLibert about your comment on the tests, have a look at https://github.com/massalabs/massa-sc-toolkit/pull/68#discussion_r1047420981

I think we can merge it and the following issue will improve that

**Review by gregLibert — APPROVED** [review]
:+1:

**Pull request merged**